lib/bob/job/otp_checker.ex

```diff
@@ -1,12 +1,14 @@
 defmodule Bob.Job.OTPChecker do
   @repo "erlang/otp"
   @linuxes ["ubuntu-14.04", "ubuntu-16.04", "ubuntu-18.04", "ubuntu-20.04", "ubuntu-22.04"]
+  @arches ["amd64", "arm64"]
 
   def run(_type) do
     for linux <- @linuxes,
+        arch <- @arches,
         {ref_name, ref} <- Bob.GitHub.diff(@repo, "builds/otp/#{linux}"),
```

Comment:
This diff won't work since it currently only diffs the existing amd64 builds. We need to track the architectures separately in different files https://github.com/hexpm/bob/blob/main/lib/bob/repo.ex#L5-L10.

Reply:
I suggest using the file path `builds/otp/#{arch}/#{linux}"` going forward, before the deploy we will move the files in the bucket and we will do a path rewrite on the CDN to ensure the existing paths without arch doesn't break.